api/handlers.go

```diff
@@ -696,7 +623,7 @@ func (api *API) userPubKeyRegisterGET(w http.ResponseWriter, req *http.Request,
 		return
 	}
 	uu := &database.UnconfirmedUserUpdate{
-		Sub:         sub,
+		Sub:         u.Sub,
 		ChallengeID: ch.ID,
 		ExpiresAt:   ch.ExpiresAt.Truncate(time.Millisecond),
```

Comment:
It doesn't matter, tbh. If we don't truncate it here MongoDB will truncate it when we store it. I started truncating timestamps everywhere on creation, so unit tests don't fail because of the differences.